x-pack/plugins/triggers_actions_ui/public/application/hooks/use_load_rule_alerts_aggregations.ts

```diff
@@ -146,51 +142,109 @@ export async function fetchRuleAlertsAggByTimeRange({
               {
                 range: {
                   '@timestamp': {
-                    // When needed, we can make this range configurable via a function argument.
                     gte: 'now-30d',
                     lt: 'now',
                   },
                 },
               },
+              {
+                bool: {
+                  should: [
+                    {
+                      term: {
+                        'kibana.alert.status': 'active',
+                      },
+                    },
+                    {
+                      term: {
+                        'kibana.alert.status': 'recovered',
+                      },
+                    },
+                  ],
+                },
+              },
             ],
           },
         },
         aggs: {
-          filterAggs: {
+          total: {
             filters: {
               filters: {
-                alert_active: { term: { 'kibana.alert.status': 'active' } },
-                alert_recovered: { term: { 'kibana.alert.status': 'recovered' } },
+                totalActiveAlerts: {
+                  term: {
+                    'kibana.alert.status': 'active',
+                  },
+                },
+                totalRecoveredAlerts: {
+                  term: {
+                    'kibana.alert.status': 'recovered',
+                  },
+                },
+              },
+            },
+          },
+          statusPerDay: {
+            date_histogram: {
+              field: '@timestamp',
+              fixed_interval: '1d',
+              extended_bounds: {
+                min: 'now-30d',
+                max: 'now',
               },
             },
             aggs: {
-              status_per_day: {
-                date_histogram: {
-                  field: '@timestamp',
-                  fixed_interval: '1d',
+              alertStatus: {
+                terms: {
+                  field: 'kibana.alert.status',
                 },
               },
             },
           },
         },
       }),
     });
-    const active = res?.aggregations?.filterAggs.buckets.alert_active?.doc_count ?? 0;
-    const recovered = res?.aggregations?.filterAggs.buckets.alert_recovered?.doc_count ?? 0;
+
+    const active = res?.aggregations?.total.buckets.totalActiveAlerts?.doc_count ?? 0;
+    const recovered = res?.aggregations?.total.buckets.totalRecoveredAlerts?.doc_count ?? 0;
+    // Total alerts count in the last 30 days per status
+    const totalAlerts = active + recovered;
+
     const alertsChartData = [
```

Comment:
nit -> will be nice to create a function and have some basic unit test to do this formatting

Reply:
To be covered in https://github.com/elastic/kibana/issues/136943